korman/ui/ui_menus.py

```diff
@@ -40,8 +40,25 @@ def build_plasma_menu(self, context):
     self.layout.separator()
     self.layout.menu("menu.plasma_add", icon="URL")
 
+
+class korman_help_menu(bpy.types.Menu):
```

Comment:
The class name is not consistent with PEP8, and we need to inherit from `PlasmaMenu`.